src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -187,6 +187,8 @@ function MoneyRequestReportPreviewContent({
     const shouldShowRTERViolationMessage = numberOfRequests === 1 && hasPendingUI(lastTransaction, lastTransactionViolations);
     const shouldShowOnlyPayElsewhere = useMemo(() => !canIOUBePaid && getCanIOUBePaid(true), [canIOUBePaid, getCanIOUBePaid]);
 
+    const [reportAttributes] = useOnyx(ONYXKEYS.DERIVED.REPORT_ATTRIBUTES, {canBeMissing: true, selector: (c) => c?.reports});
```

Comment:
No inline selectors, can you please update this in vein of https://github.com/Expensify/App/issues/70272

Reply:
Does the `getMoneyReportPreviewName` need all the report information or we should only recompute it when only the specific properties we need change? feels like it is wasteful to recompute this with any report attributes change cc @TMisiukiewicz